Lib/test/test_patma.py

```diff
@@ -2760,6 +2760,120 @@ def test_patma_255(self):
         self.assertEqual(y, 1)
         self.assertIs(z, x)
 
+    def test_patma_256(self):
```

Comment:
I suppose it's the established pattern in this file, but I'm not sure we should be numbering all of these tests instead of giving them names that reflect what they're testing...